flask_github.py

```diff
@@ -24,6 +24,45 @@
 null_handler = logging.NullHandler()
 _logger.addHandler(null_handler)
 
+#: MIME type of JSON
+JSON_MIMETYPE = 'application/json'
+
+#: Generic (default) MIME type if response headers are not set
+GENERIC_MIMETYPE = 'application/octet-stream'
```

Comment:
While I might agree that `GENERIC_MIMETYPE` is unnecessary, we do use the string `application/json` more than a few times, and I wanted to prevent typo errors.

Maybe the issue is the name? Is `JSON_MIMETYPE` not clear that it would be `application/json`?